docs/documentation/quartz-3.x/how-tos/reschedule-jobs.md

```diff
@@ -0,0 +1,49 @@
+# Reschedule Jobs
+
+A few ways to approach a need to reschudel a job.
+
+## Manually Retry
+
+When a Quartz job is running, and an unhandled exception escapes the `IJob`, the Quartz system will mark the job in an error state. This would then allow you to reschedule the job using any method that would be work for your system. 
```

Comment:
Maybe should mention the pattern of `try`-`catch` handling exceptions inside the job and then throwing job `JobExecutionException` to notify what should be done. This was more easily enforced in Java version's API as only `JobExecutionException` was allowed to be get out of the job. Exception allows you to signal whether retry should be immediately tried or whether to unschedule all triggers (this job cannot succeed ever).